src/Entity/Tag.php

```diff
@@ -12,6 +12,7 @@
 namespace App\Entity;
 
 use Doctrine\ORM\Mapping as ORM;
+use JsonSerializable;
```

Comment:
in the Symfony coding standards, we don't add use statements for classes in the global namespace